pkgs/applications/networking/versus/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
```

Comment:
Is this a go module? If yes we could use buildGoModule and avoid the deps.nix.

Reply:
Oh, it is! Rebased.